test/index.ts

```diff
@@ -31,4 +33,60 @@ describe('Onboard Guilherme Brito', () => {
       expect(res.data.data.hello).to.be.equal('Hello World');
     });
   });
+
+  describe('#create user mutation', () => {
+    afterEach(async () => {
+      await prisma.user.deleteMany();
+    });
+
+    it('should create a user with the correct informations', async () => {
+      const userInput: UserInput = {
+        birthDate: '25/04/2003',
+        email: 'teste@taqtile.com.br',
+        name: 'guilherme',
+        password: 'senha123',
+      };
+
+      const graphqlMutation = `#graphql
+        mutation CreateUser($data: UserInput!) {
+          createUser(data: $data) {
+            birthDate
+            email
+            id
+            name
+          }
+        }`;
+
+      const graphqlMutationRequestBody = {
+        operationName: 'CreateUser',
+        query: graphqlMutation,
+        variables: { data: userInput },
+      };
+
+      const {
+        data: {
+          data: { createUser },
+        },
+      } = await axios.post(LOCAL_SERVER_URL, graphqlMutationRequestBody);
+
+      const dbUser = await prisma.user.findUnique({
+        where: {
+          email: 'teste@taqtile.com.br',
+        },
+      });
+
+      expect(dbUser.name).to.be.equal('guilherme');
+      expect(dbUser.email).to.be.equal('teste@taqtile.com.br');
+      expect(dbUser.birthDate).to.be.equal('25/04/2003');
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+      expect(bcrypt.compareSync('senha123', dbUser.password)).to.be.true;
```

Comment:
Tá tendo alguma unused expression aqui? 🤔

Reply:
Sim, achei estranho tambem, mas o lint no vscode ta acusando e rodando o comando do lint no terminal tambem. Deu isso aqui:
81:7  error  Expected an assignment or function call and instead saw an expression  @typescript-eslint/no-unused-expressions